temel_bilgiler/kurulumlar/web_nasıl_çalışır.md

```diff
@@ -0,0 +1,57 @@
+*   [What is a network?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+*   [What is the internet?](https://www.youtube.com/watch?v=7_LPdttKXPc&t=46s)
+*   [What is an IP address?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+*   [What is a router?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+*   [What is an ISP?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+*   [What are packets and how are they used to transfer data?](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works#packets_explained)
+*   [What is a client?](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works#clients_and_servers)
+*   [What is a server?](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works#clients_and_servers)
+*   [What is a web page?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Pages_sites_servers_and_search_engines)
+*   [What is a web browser?](https://www.youtube.com/watch?v=BrXPcaRlBqo&feature=youtu.be)
+*   [What is a search engine?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Pages_sites_servers_and_search_engines)
+*   [What is a DNS request?](https://www.youtube.com/watch?v=72snZctFFtA&t=45s)
+*   [Which browser are you currently using?](https://www.whatsmybrowser.org/)
+*   [In your own words, explain what happens when you run a search on google.com.](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Pages_sites_servers_and_search_engines)
+
+### Ek Kaynaklar
+
+Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
+
+*   Security Now Podcast'in *Web nasıl çalışır* serisinin [Bölüm 1](https://twit.tv/shows/security-now/episodes/25?autostart=false) (8:30 civarında başlar) ve [Bölüm 2](https://twit.tv/shows/security-now/episodes/26?autostart=false)'sini dinleyin.
+*   [A packet's journey](https://www.youtube.com/watch?v=ewrBalT_eBM&feature) izleyin.
+*   [Tubes: Behind the Scenes at the Internet](https://www.amazon.co.uk/dp/B007TB5SKA/ref=dp-kindle-redirect?_encoding=UTF8&btkr=1) kitabını satın almayı düşünün.
+*   Biraz daha okumak istiyorsanız, LaunchSchool'daki [Introduction to HTTP](https://launchschool.com/books/http) kitabına göz atabilirsiniz. Bu kitap aynı zamanda geliştirici araçları ve güvenlik gibi müfredatta daha sonra ele alınacak bazı konulara da değinmektedir. Ek olarak, gelecekte faydalı bulabileceğiniz HTTP araçları hakkında bilgi edinebilirsiniz.
+*   Fiber optiğin verileri uzun mesafelerde iletmek için ışığı nasıl kullandığını ve entegre fotonik ile sanal dünyamızı internetin ötesine nasıl genişlettiğini keşfedin. [The hidden network that makes the internet possible](https://youtu.be/er3v4PVNQqE).
```

Comment:
fix: * Fiber optiğin, uzun mesafelere verileri iletmek için ışığı nasıl kullandığını ve entegre fotonik ile sanal dünyamızı internetin ötesine nasıl taşıdığını keşfedin. [The hidden network that makes the internet possible](https://youtu.be/er3v4PVNQqE).